src/rmp/test/cpp/TestAbc.cc

```diff
@@ -94,10 +95,12 @@ class AbcTest : public ::testing::Test
     // Assumes module name is "top" and clock name is "clk"
     sta::dbNetwork* network = sta_->getDbNetwork();
     ord::dbVerilogNetwork verilog_network;
+    sta::VerilogReader* verilog_reader = nullptr;
     verilog_network.init(network);
-    ord::dbReadVerilog(file_name.c_str(), &verilog_network);
+    ord::dbReadVerilog(file_name.c_str(), &verilog_network, verilog_reader);
     ord::dbLinkDesign(top.c_str(),
                       &verilog_network,
+                      verilog_reader,
```

Comment:
verilog_reader is deleted by this call so it would be good to null the pointer.

Reply:
Done.